.vimrc

```diff
@@ -0,0 +1,10 @@
+filetype plugin indent on
```

Comment:
I'm not using this setting, I don't like it. Please remove it. Other settings may sense.